### PR TITLE
Update error code and message for missing activity task.

### PR DESCRIPTION
**Pull request by jbreiding**
**What changed?**
Error code and message for missing activity task

**Why?**
A bit more clear on reason for failure and direction for investigation
#2538 

**How did you test it?**
unit tests

**Potential risks**
None

**Is hotfix candidate?**
No

### CODE REVIEW

File: service/history/historyEngine.go [hunk 1]
```diff
@@ -2850,11 +2850,11 @@ func getScheduleID(
 ) (int64, error) {
 
 	if activityID == "" {
-		return 0, serviceerror.NewInvalidArgument("Neither ActivityID nor ScheduleID is provided")
+		return 0, serviceerror.NewInvalidArgument("invalid ActivityID provided")
```
Comment (wxing1292) [inline service/history/historyEngine.go:2853]: `Invalid activity ID` ?

**Review by wxing1292 — APPROVED** [review]

File: service/history/historyEngine.go (hunk 1 above)
Comment (jbreiding) [reply service/history/historyEngine.go:2853]: Okay, that is a fair point.
idiomatic go says errors should be lower cased so they can be `fmt'ed` into a logging line which may be sentenced cased.

I did depart from what was there before, happy to change though to keep consistency.

Thoughts?

File: service/history/historyEngine.go [hunk 2]
```diff
@@ -2850,11 +2850,11 @@ func getScheduleID(
...
 	}
 	activityInfo, ok := mutableState.GetActivityByActivityID(activityID)
 	if !ok {
-		return 0, serviceerror.NewInvalidArgument("Cannot locate Activity ScheduleID")
+		return 0, serviceerror.NewNotFound(fmt.Sprintf("cannot locate active task for ActivityID %s, check workflow execution history for more details", activityID))
```
Comment (yiminc) [inline service/history/historyEngine.go:2857]: suggest: cannot find pending activity with ActivityID %s

File: service/history/historyEngine.go (hunk 1 above)
Comment (yiminc) [reply service/history/historyEngine.go:2853]: I would just say `activityID cannot be empty`

**Review by yiminc — APPROVED** [review]

**Pull request merged**